src/views/clusteroverview/OverviewTab/resource-counts-card/ResourceCountsCard.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+
+import { NodeModel } from '@kubevirt-ui/kubevirt-api/console';
+import NetworkAttachmentDefinitionModel from '@kubevirt-ui/kubevirt-api/console/models/NetworkAttachmentDefinitionModel';
+import VirtualMachineModel from '@kubevirt-ui/kubevirt-api/console/models/VirtualMachineModel';
+import { useIsAdmin } from '@kubevirt-utils/hooks/useIsAdmin';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { getResourceUrl } from '@kubevirt-utils/resources/shared';
+import { Card, Grid, GridItem } from '@patternfly/react-core';
+
+import useInventoryCounts from './hooks/useInventoryCounts';
+import ResourceCountItem from './ResourceCountItem';
+
+import './ResourceCountsCard.scss';
+
+const ResourceCountsCard: React.FC = () => {
+  const { t } = useKubevirtTranslation();
+  const isAdmin = useIsAdmin();
+  const countsData = useInventoryCounts();
```

Comment:
i think u can simply deconstruct here vms, temples, etc.. instead of countsData?....

Reply:
Fixed!